05_data_model_extensions.sql

```diff
@@ -72,17 +65,32 @@ UPDATE qgep_vl.channel_usage_current SET order_usage_current=9 WHERE code = 5322
 -- TABLE wastewater_node
 ALTER TABLE qgep_od.wastewater_node ADD COLUMN _usage_current integer;
 COMMENT ON COLUMN qgep_od.wastewater_node._usage_current IS 'not part of the VSA-DSS data model
-added solely for QGEP
+added solely for TEKSI wastewater
 has to be updated by triggers';
 ALTER TABLE qgep_od.wastewater_node ADD COLUMN _function_hierarchic integer;
 COMMENT ON COLUMN qgep_od.wastewater_node._function_hierarchic IS 'not part of the VSA-DSS data model
-added solely for QGEP
+added solely for TEKSI wastewater
 has to be updated by triggers';
 ALTER TABLE qgep_od.wastewater_node ADD COLUMN _status integer;
 COMMENT ON COLUMN qgep_od.wastewater_node._status IS 'not part of the VSA-DSS data model
-added solely for QGEP
+added solely for TEKSI wastewater
 has to be updated by triggers';
 
+-- label extensions
+CREATE TABLE IF NOT EXISTS qgep_od.labels
```

Comment:
https://github.com/teksi/Home/wiki/TEKSI-Developer-Guide#custom-fields
_labels instead of labels as it is not a VSA-DSS class

Reply:
Or move it to qgep_sys?